tables.js

```diff
@@ -5,7 +5,10 @@
 //
 //
 
-
+var default_macros = {"Physician-roll":"?{Damage Type modifier|Minor cut,30|Serious cut,20|Grevious cut,10|Minor stab,25|Serious stab,15|Grevious stab,5|Minor bruise,30|Serious blunt - fracture,20|Grevious blunt - crush,10|Bleeding wound,50}",
```

Comment:
Let's rename this "treatement bonus" and argument the text with the proper treatment, e.g. "Minor cut - Clean & dress,30"